modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/NewApplication/NewApplicationAuto/index.tsx

```diff
@@ -112,17 +102,55 @@ const NewApplicationAuto = ({children}: DriverInfoProps) => {
 		if (state.currentStep === 4) {
 			redirectTo('Applications');
 		}
+
+		if (
+			state.currentStep === 2 &&
+			form[0]?.accidentCitation[0]?.value ===
+				'Citation - Driving under the influence'
+		) {
+			createOrUpdateRaylifeApplication(
+				state,
+				CONSTANTS.APPLICATION_STATUS.REJECTED
+			).then((response) => {
+				const {
+					data: {id},
+				} = response;
+
+				dispatch({payload: {id}, type: ACTIONS.SET_APPLICATION_ID});
+			});
+			dispatch({
+				payload: 5,
+				type: ACTIONS.SET_CURRENT_STEP,
+			});
+		}
+		else {
+			createOrUpdateRaylifeApplication(state, applicationStatus).then(
+				(response) => {
+					const {
+						data: {id},
+					} = response;
+
+					dispatch({payload: {id}, type: ACTIONS.SET_APPLICATION_ID});
+				}
+			);
+		}
```

Comment:
Please, remove the else condition.

```suggestion
		if (
			state.currentStep === 2 &&
			form[0]?.accidentCitation[0]?.value ===
				'Citation - Driving under the influence'
		) {
			createOrUpdateRaylifeApplication(
				state,
				CONSTANTS.APPLICATION_STATUS.REJECTED
			).then((response) => {
				const {
					data: {id},
				} = response;

				dispatch({payload: {id}, type: ACTIONS.SET_APPLICATION_ID});
			});
			
			return dispatch({
				payload: 5,
				type: ACTIONS.SET_CURRENT_STEP,
			});
		}
		
		createOrUpdateRaylifeApplication(state, applicationStatus).then(
			(response) => {
				const {
					data: {id},
				} = response;

				dispatch({payload: {id}, type: ACTIONS.SET_APPLICATION_ID});
			}
		);
		
```

Reply:
Done!